packages/validator/test/session.spec.js

```diff
@@ -310,3 +310,47 @@ test('resolve key', async () => {
     },
   })
 })
+
+test('service can not delegate access to account', async () => {
+  const account = Absentee.from({ id: 'did:mailto:web.mail:alice' })
+  // service should not be able to delegate access to account resource
+  const auth = await Delegation.delegate({
```

Comment:
I used ucanto/core `delegate` export, which I believe ends up calling this, but was small diff